src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1261,6 +1261,15 @@ bool isAllowedInitOnlySet(BoundExpression receiver)
             return false;
         }
 
+        internal static uint GetInterpolatedStringHandlerConversionEscapeScope(
+            BoundInterpolatedString interpolatedString,
+            uint scopeOfTheContainingExpression)
+        {
+            Debug.Assert(interpolatedString.InterpolationData != null);
+            var data = interpolatedString.InterpolationData.GetValueOrDefault();
+            return GetValEscape(data.Construction, scopeOfTheContainingExpression);
```

Comment:
Just to confirm we don't need to deal with `GetRefEscape` here in some cases?
The escapes rules are complicated enough that I'm having trouble reloading this on the fly...

Reply:
I don't believe so, but this is not my strongest area.

Reply:
Agree here that I *think* its ok, but I'm not 100% convinced. Can we add a work item to try and make a test that convinces us?